contrib/devtools/github-merge.sh

```diff
@@ -83,12 +83,13 @@ function cleanup() {
 
 # Create unsigned merge commit.
 (
-  echo "Merge pull request #$PULL"
+  PRTITLE=`curl -s https://api.github.com/repos/$REPO/pulls/$PULL | grep -e '  "title": ".*",'| awk -F'"' '{print $4}'`
```

Comment:
$REPO is supposed to be something like `bitcoin/bitcoin` as per [this](https://github.com/btcdrak/bitcoin/blob/master/contrib/devtools/github-merge.sh#L15-L18)